README.md

```diff
@@ -1,55 +1,76 @@
-* `when` identifies when and how often an event should happen such as "everyday at 6pm", or "every workday at 4:30pm"
+* `when` identifies when and how often an event should happen such as "everyday at 6:00 pm", or "every workday at 4:30 pm"
 * `time range` identifies a period of time such as "1 week", or "3 days"
+* `time zone` identifies the timezone corresponding to the `when` input
 
-### slash commands
+### Slash Commands
 
-* `/lj schedule [name] with [list of @hacker] [when]`
-  Schedule a new stand-up with a group of hackers
-  * e.g. "schedule artris with @dtoki, and @alireza everyday at 6am"
+* #### Schedule a new stand-up with a group of hackers
+    `/lj schedule [name] with [list of @hacker] [when] [timezone]`
+  * e.g. "schedule artris with @dtoki, and @alireza everyday at 6:00 am"
   * When there is a confilict, the hacker will not be added to the new stand-up and a warning will be displayed
-* `/lj add [me or list of @hacker] to [name]`
-  Add hackers to an existing stand-up
+  * If a stand-up with the same name already exists, the user will get asked to provide a different name
+  
+
+* #### Add hackers to an existing stand-up
+    `/lj add [me or list of @hacker] to [name]` 
   * e.g. "add @dtoki, and @alireza to artris"
-  * When there is a confilict, the hacker will not be added to the new stand-up and a warning will be displayed
-* `/lj remove [me or a list @hacker] from [name]`
-  Remove hackers from an existing stand-up
+  * When there is a conflict, the hacker will not be added to the new stand-up and a warning will be displayed
+
+* #### Remove hackers from an existing stand-up
+    `/lj remove [me or a list of @hacker] from [name]`
   * e.g. "remove me from artris"
-* `/lj move [name] to [when]`
-  Reschedule a stand-up
-  * e.g. "move artris to 7am"
-* `/lj terminate [name]`
-  Terminate a stand-up
-  * e.g. "terminate artris"
-* `lj halt [name]`
-  Nofity the bot when the team is on a break
+  * An error message will be displayed if the hacker does not belong to the specified stand-up
+
+* #### Reschedule a stand-up
+    `/lj move [name] to [when]`
+  * e.g. "move artris to everyday at 7:00 am"
+  * An warning will be displayed if the new schedule results in conflicts among participant hacker schedules
+
+* #### Temporarily halt a stand-up
+    `lj halt [name]`
   * e.g. "halt artris"
-* `lj resume [name]`
-  Nofity the bot when you want to restart the stand-ups
+
+* #### Resume a stand-up
+    `lj resume [name]`
   * e.g. "resume artris"
-* `/lj status`
-  Display info about the active stand-ups and member participation
-* `/lj [I or @hacker] will skip [name] [time range]`
-  Notify the bot before taking a break
-  * At least 8h before the stand-up
+
+* #### Terminate a stand-up
+    `/lj terminate [name]`
+  * e.g. "terminate artris"
+
+
+* #### Notify an individual's timed break for a stand-up
+    `/lj [I or @hacker] will skip [name] [time range]`
+
+  * Allowed at least 8h before the stand-up
```

Comment:
This is actually not part of the implementation yet so we should remove it for now